MergerLogic/Clients/GpkgClient.cs

```diff
@@ -673,8 +673,9 @@ public void UpdateTileMatrixTable(bool isOneXOne = false)
                 connection.Open();
                 using (var command = connection.CreateCommand())
                 {
-                    command.CommandText = $"SELECT  MAX(zoom_level) AS maxZoom FROM \"{this._tileCache}\";";
-                    maxZoom = int.Parse(command.ExecuteScalar().ToString());
+                    command.CommandText = $"SELECT MAX(zoom_level) AS maxZoom FROM \'{this._tileCache}\';";
+                    var res = command.ExecuteScalar();
+                    maxZoom = res != DBNull.Value ? int.Parse(res.ToString()) : 0;
```

Comment:
approved, check the 'Possible null reference' error

Reply:
`res` cannot be `null` here and the check is for making sure there is a value.